ui/app/routes/vault/cluster/policies/policy.js

```diff
@@ -1,36 +1,31 @@
-import { hash } from 'rsvp';
 import Route from '@ember/routing/route';
-import UnloadModelRoute from 'vault/mixins/unload-model-route';
 import { inject as service } from '@ember/service';
 
-export default Route.extend(UnloadModelRoute, {
```

Comment:
By removing this mixin the model will no longer be unloaded from the store. I think the easiest thing to do would be to convert the mixin to a base route class and then any routes that require that functionality can extend it.